Testreglar/1.2.1/App/app-1.2.1a.json

```diff
@@ -0,0 +1,167 @@
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Lydklipp er direkte/live."
+				}
+			}
+		},
+		{
+			"stegnr": "3.3",
+			"spm": "Er lydklippet et mediealternativ til tekst eller video uten lyd og tydelig merket som det?",
+			"ht": "<p>For å være mediealternativ skal lydklippet:</p><ul><li>ikke presentere mer informasjon enn det som gis via tekst eller video uten lyd og</li><li>være tydelig merket. Det vil si<ul><li>være plassert i direkte tilknytning til aktuell tekst eller video uten lyd og</li><li>gi informasjon som viser at det er et mediealternativ</li></ul></li></ul><p>Slik tester du:</p><ul><li>Undersøk om det i tilknytning til lydklippet, er en tekst eller video uten lyd, som fremstår som hovedformatet for formidling av informasjonen</li><li>Momenter som tilsier at lydklippet er et mediealternativ til tekst eller video uten lyd kan være:<ul><li>Ikon, for eksempel lyd-ikon</li><li>Lenke med lenketekst som viser til en tekst eller video uten lyd, for eksempel \"lytt til teksten\" eller \"lytt til videoen\"</li></ul></li><li>Sjekk at lydklippet ikke presenterer mer informasjon enn det som gis via teksten eller videoen uten lyd</li></ul>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "ikkjeForekomst",
+					"utfall": "Lydklipp er mediealternativ til tekst eller video uten lyd."
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.4"
+				}
+			}
+		},
+		{
+			"stegnr": "3.4",
+			"spm": "Er det en tekst visuelt plassert nær lydklippet/avspilleren som omtaler klippet?",
+			"ht": "<p>Slik tester du:</p><ul><li>Gjør en visuell inspeksjon</li><li>Sjekk om du finner et tekstalternativ nær lydklippet (på samme appside), for eksempel rett over eller under lydklippet.</li></ul><p><strong>Merk:</strong> Du skal ikke ta stilling til kvaliteten på teksten i dette steget.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.6"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.5"
+				}
+			}
+		},
+		{
+			"stegnr": "3.5",
+			"spm": "Ligger det en mekanisme i nærheten av lydklippet, som går til et tekstalternativ for klippet?",
+			"ht": "<p>Slik tester du:</p><ul><li>Gjør en visuell inspeksjon</li><li>Sjekk om det ligger mekanisme (lenke, knapp eller lignende) i nærheten av lydklippet som går til tekstalternativet<ul><li>Eksempel på plassering: rett over eller under lydklippet</li></ul></li></ul>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.6"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Lydklippet har ikke tekstalternativ, plassert nær klippet."
```

Comment:
Vurder å utvid utfall: feks: " Lydklippet har verken tekstalternativ i nærheten av klippet, eller en mekanisme som tar brukeren til et tekstalternativ for klippet"